classes/Policies.php

```diff
@@ -273,6 +273,22 @@ class Policies {
         )
     );
 
+    protected $set_descs = array(
+        'social' => 'Social Issues',
```

Comment:
This commit says "move" but they're still in mp/index.php too, rather than use this array.